packages/happo-core/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "happo-core",
+  "version": "5.0.0",
+  "description": "Command-line tool to visually diff JavaScript components",
```

Comment:
Not sure if this is important, but this might be better as "Core modules used by other happo packages" or something similar.